scripts/assets/test_file_exemptions.textproto

```diff
@@ -383,6 +383,12 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/conceptcard/C
 exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardListener.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardViewModel.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/info/TopicInfoFragmentPresenter.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/info/TopicInfoSkillItemViewModel.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/topic/info/TopicInfoStoryItemViewModel.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/topicdownloaded/TopicDownloadedActivity.kt"
```

Comment:
Add test for TopicDownloadedActivity and TopicDownloadedFragment

1. Test accessibility label for the activity
2. Test if clicking on view will open topic activity with all four tabs